README.md

```diff
@@ -23,9 +23,15 @@ Features of `react-image-gallery`
 ## Getting started
 
 ```
+npm install react
 npm install react-image-gallery
 ```
 
+or
+```
+npm install react react-image-gallery
```

Comment:
They shouldn't have to install react if they're installing this package. This should be removed.

Reply:
They have to because `react` is now a peerDependency and not direct dependency. That was the whole point of my issue and PR.

Reply:
What I'm saying is they should already have react installed prior to installing `react-image-gallery`. We can add a note that says something similar to what `react-redux` does if you like `react-image-gallery requires React 16 or later.`

https://github.com/reduxjs/react-redux